vic/drivers/shared_all/src/calc_root_fraction.c

```diff
@@ -62,7 +63,16 @@ calc_root_fractions(veg_con_struct  *veg_con,
         Zsum = 0;
         zone = 0;
 
+        max_itter = -1;
```

Comment:
Why not start at `0` and update the iterator at the end of the iteration rather than the beginning?